docs/policy/service-changes-2018.md

```diff
@@ -0,0 +1,188 @@
+
+            :::console
+            user@host $ grep grid.iu.edu /etc/yum.repos.d/osg*.repo
+
+        Replace all instances of `grid.iu.edu` with `opensciencegrid.org`.
+
+The exact date for moving the physical hosting location will be announced.
+If you experience any problems with the OSG Software repository, please contact us at
+[help@opensciencegrid.org](mailto:help@opensciencegrid.org).
+
+VOMS Admin Server
+-----------------
+
+The [OSG VOMS](https://voms.opensciencegrid.org:8443/voms/osg/user/home.action) service is used to sign VOMS attributes
+for members of the OSG VO and can respond to queries for a list of VO members.
+The [retirement ofVOMS Admin Server](/policy/voms-admin-retire) has been planned for quite some time so the OSG VOMS 
+servers will be retired.
+
+!!! info "Action item"
+    If your site accepts OSG jobs, transition your hosts to 
+    [LCMAPS VOMS authentication](http://opensciencegrid.github.io/docs/security/lcmaps-voms-authentication/).
+
+MyOSG and OIM
+-------------
+
+The [MyOSG](https://my.opensciencegrid.org/about) service provides web and REST interfaces to access information about
+OSG site topology, projects, and VOs.
+The MyOSG web interface will be retired but we will continue to offer the same REST interface.
+If you run a service that queries MyOSG:
+
+!!! info "Action items"
+    - Ensure that the services use <https://my.opensciencegrid.org> instead of <https://myosg.grid.iu.edu>
+    - Let us know what queries you’re making and why you’re making them
+
+[OIM](https://oim.opensciencegrid.org/oim/home) serves as the database for the information used by MyOSG with a web
+interface for data updates.
+The OIM web interface will be retired but we will migrate its data to a series of YAML files held in a GitHub repository.
+After OIM is retired, updates to the aforementioned data can be requested via e-mail or pull-request.
+Documentation forthcoming.
+
+!!! note
+    Please see the [OSG CA](#osg-ca) section for information regarding the OIM certificate service.
+
+The exact dates for retiring the MyOSG and OIM web interfaces will be announced.
+If you experience any problems with the OSG Software repository, please contact us at
+[help@opensciencegrid.org](mailto:help@opensciencegrid.org).
+
+
+RSV
+---
+
+The [central RSV service](https://my.opensciencegrid.org/rgcurrentstatus/index?summary_attrs_showservice=on&summary_attrs_showrsvstatus=on&summary_attrs_showfqdn=on&current_status_attrs_shownc=on&gip_status_attrs_showtestresults=on&downtime_attrs_showpast=&account_type=cumulative_hours&ce_account_type=gip_vo&se_account_type=vo_transfer_volume&bdiitree_type=total_jobs&bdii_object=service&bdii_server=is-osg&start_type=7daysago&start_date=04%2F19%2F2018&end_type=now&end_date=04%2F19%2F2018&all_resources=on&gridtype=on&gridtype_1=on&active_value=1&disable_value=1)
+is a monitoring tool that displays every site service in the OSG.
```

Comment:
that displays service status information about OSG sites that elect to provide it.